python/ql/src/Classes/SubclassShadowing/SubclassShadowing.qhelp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+When an object has an attribute that shares the same name a method on the object's class (or another class attribute), the instance attribute is 
```

Comment:
```suggestion
When an object has an attribute that shares its name with a method on the object's class (or another class attribute), the instance attribute is 
```
Something about the previous formulation looked off to me. Feel free to rewrite it in some other way.